### DISCUSSION TITLE
Does streaming UI (loading.tsx) affect Cumulative Layout Shift (CLS)?

### DISCUSSION THREAD

**Discussion by jakubhome** [vercel/next.js#73733] [Help]
### Summary

I mean for example skeletons in loading.tsx, do they have to be exactly the same dimensions as the target content or can this be omitted for CLS?

**Comment by icyJoseph** [comment]
Not by itself, but you should try to make sure that the fallback/loading, takes up about the same space as the content would.

loading.tsx, is a suspense boundary at the end of the day, same rules apply.
